### PR TITLE
Fix scraper for new table on FR page

### LINKED ISSUE #255: Python Scraper FR still fails
cc @baryluk 

```
Traceback (most recent call last):
  File "././scrape_fr.sh", line 49, in <module>
    assert m, "Can't find table"
AssertionError: Can't find table
FR -                      -       - FAILED 2020-03-27T09:23:46+01:00 Missing date. Missing cases https://www.fr.ch/covid19/sante/covid-19/coronavirus-statistiques-evolution-de-la-situation-dans-le-canton
```

### PR DESCRIPTION
**Pull request by metaodi**
Closes #255

### CODE REVIEW

**Comment by baryluk** [comment]
Could you maybe make the 2 changes:

Use `<t[dh]>` and `</t[dh]>`so it matches both td and th.

Use `<td.*?>` for non greedy match.

File: scrapers/scrape_fr.sh [hunk 1]
```diff
@@ -45,12 +45,12 @@ sc.timestamp()
 </div>
 """
 
-m = re.search(r'<table class="table table-condensed table-hover">(.*?)</table>', d, flags=re.MULTILINE | re.DOTALL)
+m = re.search(r'<table class="table table-striped.*">(.*?)</table>', d, flags=re.MULTILINE | re.DOTALL)
```
Comment (baryluk) [inline scrapers/scrape_fr.sh]: Lets use:

```
m = re.search(r'<table class="table.*?">(.*?)</table>', d, flags=re.MULTILINE | re.DOTALL)
```

**Comment by metaodi** [comment]
Good point about non-greedy match!

**Comment by metaodi** [comment]
@baryluk PTAL

**Pull request merged**